src/main/java/exception/OutOfRange.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class OutOfRange extends RuntimeException {
```

Comment:
```suggestion
public class OutOfRangeException extends RuntimeException {
```

Exception이라는 suffix를 붙여줄 필요는 없을까요?

Reply:
앗.. 좀 더 명확히 알 수 있도록 추가하겠습니다!